services/QuillLMS/engines/comprehension/lib/comprehension/comprehension/change_log.rb

```diff
@@ -0,0 +1,17 @@
+module Comprehension
+  module ChangeLog
+    def log_change(user_id, action, changed_record, explanation = nil, changed_attribute = nil, previous_value = nil, new_value = nil)
+      change_log = {
+        user_id: user_id,
+        action: Comprehension.change_log_class::COMPREHENSION_ACTIONS[action],
+        changed_record_type: changed_record.class.name,
+        changed_record_id: changed_record.id,
+        explanation: explanation,
+        changed_attribute: changed_attribute,
+        previous_value: previous_value,
+        new_value: new_value
+      }
+      Comprehension.change_log_class.create(change_log)
```

Comment:
Not the end of the world, but there's no error handling on this.  If something does go wrong, or something is invalid, this makes it look like the error will get swallowed.  Maybe that's fine for now since we don't really know what we want to do to handle errors, but something worth thinking about, maybe.